Makefile.common

```diff
@@ -981,7 +981,8 @@ ifeq ($(HAVE_PARPORT), 1)
 endif
 
 ifneq ($(findstring Win32,$(OS)),)
-   OBJ += input/drivers/winraw_input.o
+   OBJ += input/drivers/winraw_input.o \
+		  gfx/video_crt_switch.o 
```

Comment:
I think not every version of Windows may support all manners of mode switching (or RAWINPUT for that matter)... and MSVC builds do not use this file at all, instead the source file must be added in griffin/griffin.c.

Reply:
This switching method works on all windows versions from Windows XP - Windows 10 as long as CRTEmudriver is installed

Reply:
Raw input was already here